docs/msbuild/errors/msb3644.md

```diff
@@ -35,6 +35,14 @@ The first thing to check is that there are no spelling or typographical errors i
 
 The issue is that the `TargetFrameworkIdentifier` is misspelled. It should be `.NETCOREAPP`.
 
+When leveraging the `TargetFrameworks` property and multiple target frameworks, ensure that they are separated with the correct delimter `;`. MSB3644 will occur, for example, by trying to separate frameworks via the `,` delimter:
```

Comment:
Crosslink to `/dotnet/core/project-sdk/msbuild-props#targetframeworks` for this?

Reply:
Do you mean like so? Making sure that the relative link is correct here so in prod it doesn't become a dead link. I assume we shouldn't hardcode the english link https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#targetframeworks


 ``` [TargetFrameworks](/dotnet/core/project-sdk/msbuild-props#targetframeworks)```:
 [TargetFrameworks](/dotnet/core/project-sdk/msbuild-props#targetframeworks)

Reply:
Completed this and @joeloff 's suggestion to use 'using.' Thank you both for your suggestions to help improve the documentation I wrote. 👍